Formula/lanraragi.rb

```diff
@@ -0,0 +1,63 @@
+require "language/node"
+
+class Lanraragi < Formula
+  desc "Web application for archival and reading of manga/doujinshi. Lightweight and Docker-ready for NAS/servers."
+  homepage "https://github.com/Difegue/LANraragi"
+  url "https://github.com/Difegue/LANraragi/archive/v.0.6.9-homebrew.tar.gz"
```

Comment:
What's different about the homebrew version?

Reply:
Only a directory change, see https://github.com/Difegue/LANraragi/compare/v.0.6.9...v.0.6.9-homebrew